.github/workflows/opt-out-import-dev-deploy.yml

```diff
@@ -0,0 +1,38 @@
+name: opt-out-import dev deploy
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - optout/**
+      - .github/workflows/opt-out-import-dev-deploy.yml
+  workflow_dispatch:
+
+jobs:
+  test:
+    permissions:
+      contents: read
+      id-token: write
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./optout
+    environment: dev
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-go@v5
+      - name: Build opt-out-import zip file
+        run: |
+          go build -o bootstrap main.go parsers.go utils.go models.go db.go
```

Comment:
These filenames will need to be updated for BCDA optout.

Reply:
I added a make command that should build it, see here:

https://github.com/CMSgov/bcda-app/pull/947

From bcda subdirectory: `go build -o bin/opt-out-import ./lambda/optout/main.go`